Add a convenient way of constructing the operation and retrieving it.

metlos opened the pull request.

metlos committed:
Add a convenient way of constructing the operation and retrieving it.

jpkrohling commented:
Looks good in general, but could you please:
- write a test
- add an example to the readme
- bonus point: add a link on the readme pointing to the wiki page describing the roles on Wildfly's side (perhaps replacing the link to the blog post linked there already)

metlos committed:
Merge remote-tracking branch 'upstream/master' into make-and-retrieve-operation

metlos committed:
Adding a simple test for OperationService.Setup#make()

metlos committed:
Renamed OperationService.Setup#commit() to persist() to better reflect
what it is doing.

An explicit EntityManager commit is still necessary after the setups have
been persisted.

metlos committed:
Updating README with an example usage of OperationService.Setup#make()

and adding a link to the Wfly9 roles documentation.

jpkrohling commented:
Squashed and merged via CLI.
